examples/nextjs/src/utils/dotcmsClient.js

```diff
@@ -0,0 +1,32 @@
+import { dotcmsClient } from "@dotcms/client";
+
+// Client for content fetching
+export const client = dotcmsClient.init({
+    dotcmsUrl: process.env.NEXT_PUBLIC_DOTCMS_HOST, // 'http://localhost:8080'
+    authToken: process.env.NEXT_PUBLIC_DOTCMS_AUTH_TOKEN,
+    siteId: "59bb8831-6706-4589-9ca0-ff74016e02b2",
+    requestOptions: {
+        // In production you might want to deal with this differently
+        cache: "no-cache",
+    }
+});
+
+
+/**
+ * Get request params
+ *
+ * @param {*} { params, searchParams }
+ * @return {*}
+ */
+export const getRequestParams = ({ params, searchParams, defaultPath }) => {
```

Comment:
@rjvelazco Maybe this function could be a utility within the library?

Reply:
The user should determine how they want to build the request parameters based on their needs.
 
 For example, their `queryParams` can be named `customLanguageId` instead of `language_id`.

Reply:
That said, we can export a `getRequestParams` default function but it is another function to support. Thoughts @fmontes @KevinDavilaDotCMS ?

Reply:
If we can make the function to work with Angular, Next.js and everything else... we can move it to the lib.

Reply:
Maybe it should be another ticket though